docs/modules/devices/pages/local-devices/set-up-device-for-apple-pay.adoc

```diff
@@ -0,0 +1,72 @@
+= Set up a device for Apple Pay
+:navtitle: Set up Apple Pay
+
+Learn how to set up _Confirm with AssistiveTouch_ feature on an iOS/iPadOS device to xref:manual-testing:local-devices/use-apple-pay-in-manual-session.adoc[use Apple Pay].
+
+== Requirements
+
+* Access to the physical device to double-click the side button during setup of AssistiveTouch. If the device is hosted by Kobiton, submit a support ticket to assist in the setup process.
+
+* A passcode set on the device. There are two options:
+
+** A passcode can be set *and must be removed* after the setup. If the passcode is not removed after the setup, the device *will not be able to come back online* without physically inputting the passcode to unlock it.
+
+** Setup Kobiton passcode support for the devices. For Private devices, discuss passcode requirements with your Customer Success Manager who will work with support to set up passcodes. For Local devices hosted at the customer location, submit a support ticket to request passcode setup documentation for your deployment type.
+
+* [Optional] Use a link:https://support.kobiton.com/hc/en-us/articles/360055588272-Device-Cleanup[Cleanup Policy] for the devices to be tested that does not reset the device settings to prevent AssistiveTouch from turning off for the next session. Otherwise, the tester will need to turn on AssistiveTouch for each session that it is needed:
```

Comment:
Only one change - then you can publish. Please link to https://docs.kobiton.com/devices/create-a-device-cleanup-policy instead of the support site. If we need to update the docs page with more info from support, we can do that - but need to keep them within the docs site. Thanks!

Reply:
Updated. Latest preview link: http://10.2.7.208/devices/local-devices/set-up-device-for-apple-pay